scripts/generate.py

```diff
@@ -0,0 +1,45 @@
+import os
```

Comment:
Please use pathlib instead of `os`. You can see the equivalent functions of `os` [here](https://docs.python.org/fr/3/library/pathlib.html#correspondence-to-tools-in-the-os-module)